ext/legacycustomsearches/CRM/Contact/Form/Search/Custom/DateAdded.php

```diff
@@ -366,10 +366,10 @@ public function templateFile() {
    */
   public function setTitle($title) {
     if ($title) {
-      CRM_Utils_System::setTitle($title);
+      $this->setTitle($title);
```

Comment:
This will cause a loop

Reply:
I've moved title changes for legacycustomsearches to https://github.com/civicrm/civicrm-core/pull/21375 and taken them out of this PR